### PR TITLE
Fix Android EN bridge 

### PR DESCRIPTION
**Pull request by henrytao-me**
This PR fixes Android ExposureNotification.start doesn't handle `startResolutionForResult`.

Demo video https://drive.google.com/file/d/1QPR_z0houVS8-O7Haq9TIc2hdCN_j19H/view?usp=sharing

Resolves https://github.com/CovidShield/mobile/issues/115

## How to test?
- Run react native packager `yarn start`.
- In terminal, make sure you one device connected to computer, run `adb reverse tcp:8081 tcp:8081`.
  + If you have issue, run `adb kill-server && adb start-server`.
- Open project in Android Studio and build to real device. 
- Go through onboarding flow. 
- Expect to see Exposure Notification dialog.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,5 +1,5 @@
 APP_ID_IOS=app.covidshield
-APP_ID_ANDROID=app.covidshield
+APP_ID_ANDROID=com.google.android.apps.exposurenotification
```
Comment (henrytao-me) [inline .env:2]: Note: for easy testing, I figure we could use Google Sample Application Id here instead of CovidShield.

Comment (jordanAtShopify) [reply .env:2]: Not sure we should commit this

Comment (henrytao-me) [reply .env:2]: I would say yes but if we agree not to, I can add some docs to the README

Comment (jordanAtShopify) [reply .env:2]: Ya, personally I think we should keep the APP_ID consistent and add a section for testing on Android with the example bundle id in README

Comment (henrytao-me) [reply .env]: Added to README

**Comment by p0ns** [comment]
Been hitting my head for a few hours, just recompiled the last version from master, and I can't get the message to display like in your video. 

I'm testing on a pixel 2, last play services and updates

https://drive.google.com/file/d/1-XSqAri_m3PoSTgi5E6NBzkrswEBdDe-/view?usp=sharing

I made sure to have TEST_MODE=false and APP_ID_ANDROID=com.google.android.apps.exposurenotification and no other changes but no dice. 

Any other things I could try?

**Comment by henrytao-me** [comment]
Exception 17 means your email is not whitelisted by Google to use Exposure Notifications in development. You might need to contact your connection for beta access.

**Pull request merged**